mllib/src/main/scala/org/apache/spark/ml/classification/GBTClassifier.scala

```diff
@@ -45,35 +45,46 @@ import org.apache.spark.sql.types.DoubleType
  * Note: Multiclass labels are not currently supported.
  */
 @Experimental
-final class GBTClassifier(override val uid: String)
+@Since("1.4.0")
```

Comment:
It's really minor but can you reorder these annotations like other classes in ml/mllib?